### PR TITLE
whatsapp issue

### PR DESCRIPTION
**Pull request by hongjk1029**
whatsapp, whatsapp in car details, changes of web title

### CODE REVIEW

File: frontend/src/pages/customer/CarDetails.jsx [hunk 1]
```diff
@@ -43,6 +43,10 @@ const CarDetails = () => {
       setImages(temp_images);
       setBrand(brand_name);
     });
+  } 
+  function navigateToWhatsApp() {
+    // window.location.href = 'https://api.whatsapp.com/send?phone=60123160808';
+    window.open('https://api.whatsapp.com/send?phone=60123160808', '_blank');
```
Comment (daniellim051000) [inline frontend/src/pages/customer/CarDetails.jsx:49]: @hongjk1029 @zheminlow for the phone number. put this under .env

File: frontend/src/components/Header/Header.jsx [hunk 2]
```diff
@@ -122,9 +122,13 @@ const Header = ({ isLogin, setLogin}) => {
...
         <header className="header">
           {NotAdminPage() && (
-            <Helmet>
-              <script defer src="https://widget.tochat.be/bundle.js?key=76872d3b-f925-4359-b7c1-5f94ef9dbe95" />
-            </Helmet>
+            // <Helmet>
+            //   <script defer src="https://widget.tochat.be/bundle.js?key=76872d3b-f925-4359-b7c1-5f94ef9dbe95" />
+            // </Helmet>
+                    <Helmet>
+                      <script defer src="https://widget.tochat.be/bundle.js?key=d6bff245-9f32-4ecf-aa8a-12c0418bee9d"/>
```
Comment (daniellim051000) [inline frontend/src/components/Header/Header.jsx:129]: I can assume this is a private key also right? @hongjk1029. put this under .env

File: frontend/src/components/Footer/Footer.jsx [hunk 3]
```diff
@@ -176,8 +176,9 @@ const Footer = ({isLogin}) => {
             </Col>
           </Row>
         </Container>
-        <Helmet> <script defer src="https://widget.tochat.be/bundle.js?key=76872d3b-f925-4359-b7c1-5f94ef9dbe95"></script></Helmet>
-       
+        {/* <Helmet> <script defer src="https://widget.tochat.be/bundle.js?key=76872d3b-f925-4359-b7c1-5f94ef9dbe95"></script></Helmet> */}
+        <Helmet><script defer src="https://widget.tochat.be/bundle.js?key=d6bff245-9f32-4ecf-aa8a-12c0418bee9d"></script></Helmet>
```
Comment (daniellim051000) [inline frontend/src/components/Footer/Footer.jsx:180]: put the key under .env

**Review by daniellim051000 — CHANGES_REQUESTED** [review]

File: frontend/src/components/Footer/Footer.jsx (hunk 3 above)
Comment (hongjk1029) [reply frontend/src/components/Footer/Footer.jsx:180]: OK I will update this later

**Review by daniellim051000 — APPROVED** [review]

**Pull request merged**